api/src/pcapi/alembic/versions/20240212T173635_daeeeb2555c8_make_placeId_nullable_in_google_places_info.py

```diff
@@ -0,0 +1,22 @@
+"""
+Make placeId nullable in google_places_info
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "daeeeb2555c8"
+down_revision = "e8da34054129"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.alter_column("google_places_info", "placeId", existing_type=sa.TEXT(), nullable=True)
+
+
+def downgrade() -> None:
+    op.alter_column("google_places_info", "placeId", existing_type=sa.TEXT(), nullable=False)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏ça, ça doit pas marcher
A priori on ne fait pas de downgrade en prod, donc ça ne se ressentira pas (et on est peu à en faire en local)
Mais si tu passes une colonne en non nullable, il faut remplir les valeurs nulles, sinon la migration va pas passer

Reply:
Je n'ai pas fait comme j'ai dit, j'ai décidé de suprimer les lignes avec la valeur null comme ça c'est compatible avec le code d'avant, et dans le quel à la base ne rajoutait pas ces lignes là